docs/source/architecture.rst

```diff
@@ -0,0 +1,143 @@
+Drivers Atlas Testing Architecture
+==================================
+
+The Drivers Atlas Testing project validates the behavior of drivers against
+Atlas clusters. For example, a test could be to perform queries or writes
+in a loop while Atlas performs a maintenance operation, and assert that all
+queries and writes complete successfully.
+
+The project consists of the following principal components:
+
+- Astrolabe - the orchestration utility.
+- :doc:`Workload executors <spec-workload-executor>` - wrappers providing
+  a common interface to each driver's unified test runner.
+- Each driver's unified test runner.
```

Comment:
Is this really a part of this project? I think not - it is specified and implemented separately and only consumed by the workload executor. Instead, you can rephrase the `Workload executor` component:

```
- :doc:`Workload executors <spec-workload-executor>` - wrappers around each driver's 
   unified test runner that provide a common interface for consumption by Astrolabe.
```

Reply:
The goal of the architecture overview document is to provide something that a newcomer to DAT might read to get an introduction to the entire system. I agree that technically unified test runner isn't part of DAT, but without mentioning it in some way I feel that the reader would be confused/mislead as to what the components are.